cli/network.go

```diff
@@ -354,6 +355,10 @@ func (n *NetworkListCommand) runNetworkList(args []string) error {
 		return err
 	}
 
+	sort.Slice(respNetworkResource, func(i, j int) bool {
```

Comment:
also check this logic in network integration test case?

Reply:
Done.PTAL @ZYecho